spec/swagger_helper.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.configure do |config|
+  # Specify a root folder where Swagger JSON files are generated
+  # NOTE: If you're using the rswag-api to serve API descriptions, you'll need
+  # to ensure that it's configured to serve Swagger from the same folder
+  config.swagger_root = Rails.root.join('swagger').to_s
+
+  # Define one or more Swagger documents & provide global metadata for each one
+  # When you run the 'rswag:specs:swaggerize' rake task, the complete Swagger
+  # will be generated at the provided relative path under swagger_root
+  # By default, the operations defined in spec files are added to the first
+  # document below. You can override this behavior by adding a swagger_doc tag
+  # to the root example_group in your specs, e.g. describe '...', swagger_doc:
+  # 'v2/swagger.json'
+  config.swagger_docs = {
+    'v1/swagger.json' => {
+      openapi: '3.0.1',
+      info: {
+        title: 'API V1',
+        version: 'v1'
+      },
+      paths: {},
+      servers: [
+        {
+          url: 'https://{defaultHost}',
```

Comment:
debería ser http en lugar de https

Reply:
Actually, I think it's better to avoid this defaultHost variable and go directly to the route (I modified it already)